app/src/main/java/com/example/fashi_shop/model/ILoadMore.java

```diff
@@ -1,4 +1,4 @@
-package com.example.fashi_shop.Model;
+package com.example.fashi_shop.model;
 
 public interface ILoadMore {
     void LoadMore(int allItem);
```

Comment:
I detect that this code is problematic. According to the [Bad practice (BAD_PRACTICE)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#bad-practice-bad-practice), [Nm: Method names should start with a lower case letter (NM_METHOD_NAMING_CONVENTION)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#nm-method-names-should-start-with-a-lower-case-letter-nm-method-naming-convention).
Methods should be verbs, in mixed case with the first letter lowercase, with the first letter of each internal word capitalized.